config/initializers/session_store.rb

```diff
@@ -1,9 +1,16 @@
 service_url = ENV['SERVICE_URL']
 SERVICE_DOMAIN = (service_url) ? URI.parse(service_url).host : nil
+timeout_time = ENV['TIMEOUT_TIME']
```

Comment:
@SteveMarshall I've currently got three apps on my machine that respond to SESSION_TIMEOUT, each with different requirements.  I'd prefer we either set it in a PVB specific dotenv file or namespace it to the app.